packages/shared/lib/services/sync/run.service.ts

```diff
@@ -55,6 +56,7 @@ export interface SyncRunConfig {
     integrationService: IntegrationServiceInterface;
     recordsService: RecordsServiceInterface;
     dryRunService?: NangoProps['dryRunService'];
+    orchestratorClient: OrchestratorClientInterface;
```

Comment:
at this point I am not sure I am not making things worse 😢 
Having all those "delegates" is a clear sign the level of abstraction is now very wrong